src/AppInstallerCommonCore/Archive.cpp

```diff
@@ -1,79 +1,93 @@
-// Copyright (c) Microsoft Corporation.
-// Licensed under the MIT License.
-#include "pch.h"
-#include "Public/winget/Archive.h"
-
-// TODO: Move include statement to pch.h and resolve build errors
-#pragma warning( push )
-#pragma warning ( disable : 4189 4244 26451 )
-#include <pure.h>

```

Comment:
IF we do decide to go with AMSI instead of PureLib, there are a bunch other places we have to remove it from. We have the source code at `src/PureLib/`, all the references in `AdditionalIncludeDirectories` and `AdditionalDependencies`, and removing the project from the solution. There's also the some housekeeping to do with the cgmanifest and the notice file